### PR TITLE
Add TDVT test string.contains.null

### PR DESCRIPTION
**Pull request by danliusf**
Add string.contains.null to TDVT. Have confirmed that having %null% values returned is the expected behavior (unfortunately).

python -m tdvt.tdvt run-pattern hyper --exp exprtests/standard/setup.string.contains.null.txt --tdp cast_calcs.hyper.tds

### CODE REVIEW

**Comment by danliusf** [comment]
@lukewrites I've finished gsub proof with all passing results - https://teamcity.prod.tableautools.com/viewModification.html?modId=470454&personal=true&tab=vcsModificationBuilds. Can I merge to master now?

File: tdvt/tdvt/exprtests/standard/expected.setup.string.contains.null.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<results>
+  <test name='CONTAINS([str1], Null)'>
+    <sql>
+      SELECT TABLEAU.CONTAINS(&quot;Extract&quot;.&quot;str1&quot;,null::text) AS &quot;TEMP(Test)(1991836379)(0)&quot;&#10;FROM &quot;Extract&quot;.&quot;Extract&quot; &quot;Extract&quot;&#10;GROUP BY 1&#10;    </sql>
+    <query-time>
+      24.000&#10;    </query-time>
```
Comment (lukewrites) [inline tdvt/tdvt/exprtests/standard/expected.setup.string.contains.null.txt:6]:
```suggestion
```
We need to remove the `<sql>` and `<query-time>` tags.

File: tdvt/tdvt/version.py [hunk 2]
```diff
@@ -1,2 +1,3 @@
-__version__ = '2.6.8'
+__version__ = '2.6.9b1'
```
Comment (lukewrites) [inline tdvt/tdvt/version.py:1]:
```suggestion
__version__ = '2.6.9'
```Looks like this will go into artifactory as 2.6.9

**Review by lukewrites — DISMISSED** [review]
Two small changes then good to go.

**Review by rosswbrown — APPROVED** [review]

**Pull request merged**